<?xml version='1.0'?>
<sdf version="1.6">
  <model name="crack_5">
    <static>true</static>
    <link name="link">
      <collision name="collision">
        <geometry>
          <box>
            <size>1 1 1</size>
          </box>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <scale>0.3 0.3 0.3</scale>
            <uri>model://crack_5/crack_5.dae</uri>
          </mesh>
        </geometry>
      </visual>
    </link>
  </model>
</sdf>
